src/test/compile-fail/consts/const-err3.rs

```diff
@@ -7,13 +7,13 @@ fn black_box<T>(_: T) {
 
 fn main() {
     let b = 200u8 + 200u8 + 200u8;
-    //~^ ERROR const_err
```

Comment:
I am not entirely sure of the purpose of this test, it looks entirely like a duplicate of `const-err2.rs`...

Reply:
Iirc I duplicates this because there was some abort-if-errors call that caused some errors not to get emitted

Reply:
So, can I remove it?

Reply:
As long as both tests have the same diagnostics emitted at this time, yes